tests/lean/run/structWith_scott.lean

```diff
@@ -0,0 +1,1322 @@
+/- This file reproduces in pure Lean a Mathlib timeout occuring in typeclass synthesis when the
+structure elaboration uses an extra eta expansion at the top level lean4#2451 -/
```

Comment:
Hard to understand the intent when the comment just ends with a link. "As reported in"?